app/sentry.js

```diff
@@ -0,0 +1,21 @@
+import * as Sentry from '@sentry/browser';
+import { Ember } from '@sentry/integrations/esm/ember';
+
+import config from './config/environment';
+
+export function init() {
+  let integrations = [];
+  if (config.environment === 'production') {
+    integrations.push(new Ember());
+  }
+
+  let isProd = location.hostname === 'crates.io';
+  let isStaging = location.hostname === 'staging.crates.io';
+  let environment = isProd ? 'production' : isStaging ? 'staging' : 'unknown';
```

Comment:
I'm thinking about more robust ways to make this work, but not much came to mind :(

Reply:
we could use an env var for this, if we can set one that is available during the build process on heroku

Reply:
We could possibly check if `HEROKU_APP_NAME` contains `staging` (the variables should be available to the buildpack, but I don't know if the nodejs buildpack adds them to the environment). This would have the slight benefit of working in my personal staging environment, though I have no plans to enable Sentry there at this time so I'm fine with this check too.